frontend/src/features/projects/common/forms/DocumentationForm.tsx

```
@@ -1,20 +1,30 @@
+import { NoteTypes } from 'constants/noteTypes';
 import React, { Fragment } from 'react';
+import { ProjectNotes } from '..';
 import { IStepProps, IProjectTask } from '../interfaces';
 import TasksForm from './TasksForm';
 
 interface IDocumentationFormProps extends IStepProps {
   tasks: IProjectTask[];
+  showNote?: boolean;
 }
 
 /**
  * Form component of DocumentationForm.
  * @param param0 isReadOnly disable editing
  */
-const DocumentationForm = ({ isReadOnly, tasks }: IDocumentationFormProps) => {
+const DocumentationForm = ({ isReadOnly, tasks, showNote = false }: IDocumentationFormProps) => {
```

By default hide the note because this component is used on other forms.